v2/README.md

```diff
@@ -3,6 +3,14 @@ title: Azure Service Operator v2
 type: docs
 description: "Manage your Azure resources from within your Kubernetes cluster."
 ---
+
+[![Validate with live Azure resources](https://github.com/Azure/azure-service-operator/actions/workflows/live-validation.yml/badge.svg)](https://github.com/Azure/azure-service-operator/actions/workflows/live-validation.yml) [![Build Devcontainer image](https://github.com/Azure/azure-service-operator/actions/workflows/build-devcontainer-image.yml/badge.svg)](https://github.com/Azure/azure-service-operator/actions/workflows/build-devcontainer-image.yml)
+
+[![Publish docs site](https://github.com/Azure/azure-service-operator/actions/workflows/deploy-site.yml/badge.svg)](https://github.com/Azure/azure-service-operator/actions/workflows/deploy-site.yml)
```

Comment:
IMO the README.md is for users, not for us. I don't think we should put these here. They don't add clarity for our users regardless of their state (green or red).

If we want a way of notifying us all of failed actions we could do something [like this](https://www.ravsam.in/blog/send-email-notification-when-github-action-fails/) with a dummy gmail account... I don't love that but I can't see any way to actually get proactive notifications from GitHub itself. Feels like we should put in a feature ask for that (or more likely find the existing one... there is one [here](https://github.com/orgs/community/discussions/43415))

The only one that maybe makes sense to put here is `Validate with live Azure resources`, although see my other comment about hesitancy there - we used to have it here and I removed it because it's always failing. IMO we either need to make it pass and then add it or just remove it altogether.

Rest maybe could go into CONTRIBUTING.md instead if not using some email solution, as they're all for contributors?

Reply:
Badges removed from README.